packages/cms-cli/commands/hubdb.js

```diff
@@ -29,7 +32,10 @@ function configureHubDbCommand(program) {
     .version(version)
     .description('Manage HubDB tables')
     .command('create <src>', 'create a HubDB table')
-    .command('fetch <tableId> <dest>', 'fetch a HubDB table');
+    .command('import <tableId> <src>', 'import a HubDB table')
+    .command('fetch <tableId> <dest>', 'fetch a HubDB table')
```

Comment:
`import` and `fetch` are aliases? Do we want both? Using `import` makes sense to me in the context of tables/stores.